program/src/state/whitelist_v2.rs

```diff
@@ -47,16 +48,19 @@ pub struct WhitelistV2 {
     /// Authority that can freeze/unfreeze the whitelist.
     pub freeze_authority: Pubkey,
     /// Whitelist conditions that must be met to validate against the whitelist.
+    #[max_len(0)]
```

Comment:
You confirmed this produces the right size? Add a comment why this is max_len(0)

Reply:
Added comment.